python/railjson_generator/justfile

```
@@ -4,10 +4,9 @@
 help:
     @just --list --unsorted
 
-# Generate small infra
+# Update (regenerate) tests infras
 run:
-    mkdir -p small_infra_out
-    uv run scripts/small_infra.py small_infra_out
+    uv run -m railjson_generator ../../tests/data/infras ../../tests/infra-scripts/*.py
```

nitpick: Maybe this part doesn't need to change completely and we still want to create the files locally?

```suggestion
    uv run -m railjson_generator small_infra_out/ ../../tests/infra-scripts/*.py
```

---

My take is that the "local tinkering" is documented in the readme and more focused on a single script, but hard to pick the good one for everybody (or parametrize it).
And what is more frequent and universal is the update of the generated infras.

So I'd stay on that unless some new argument/vote comes up.

---

Fine with me, it was not a hard opinion anyway.